src/Client/Util.java

```diff
@@ -361,4 +365,55 @@ public static void listf(String directoryName, List<File> files) {
       }
     }
   }
+  
+  public static long username2hash(String s) {
```

Comment:
comment above that this is used for RSC127

Reply:
placed